Templates/cPP-template.adoc

```diff
@@ -45,8 +45,9 @@ Although the cPP and SD may contain minor editorial errors, the cPP is recognize
 
 For more see the http://www.commoncriteriaportal.org/[Common Criteria Portal].
 
+=== Terminology

```

Comment:
I think this should say Terms

Reply:
I checked the CC Portal and none of the cPPs seem to have this section as "Glossary" at the end of the document.
The MDF PP(as an example of a NIAP PP also uses "Glossary".

BTW ISO documents use "Terms and Definitions" and place it near the beginning  of the document

So perhaps we should stick with "Glossary"??

Reply:
I'm going to create appendices in the next update for them as laid out in the old example cPP. That may be the easiest way, there will be one for Glossary and one for Acronyms.

My main reason for having it up front is so you can see it before you have read everything else since then you can see the expected definitions first instead of having to go to the end and look at them after you are confused by a term.